sql/service_wsrep.cc

```diff
@@ -249,18 +249,26 @@ extern "C" my_bool wsrep_thd_skip_locking(const THD *thd)
 
 extern "C" my_bool wsrep_thd_order_before(const THD *left, const THD *right)
 {
+  my_bool before=FALSE;
+
   if (wsrep_thd_is_BF(left, false) &&
       wsrep_thd_is_BF(right, false) &&
-      wsrep_thd_trx_seqno(left) < wsrep_thd_trx_seqno(right)) {
```

Comment:
This could be written more concisely as
```c++
  bool before= (wsrep_thd_is_BF(left, false) &&
      wsrep_thd_is_BF(right, false) &&
      wsrep_thd_trx_seqno(left) < wsrep_thd_trx_seqno(right);
```

Reply:
Fixed.